10.1/Dockerfile.fedora

```diff
@@ -19,14 +19,12 @@ LABEL summary="MariaDB is a multi-user, multi-threaded SQL database server" \
       io.k8s.description="MariaDB is a multi-user, multi-threaded SQL database server" \
       io.k8s.display-name="MariaDB 10.1" \
       io.openshift.expose-services="3306:mysql" \
-      io.openshift.tags="database,mysql,mariadb,mariadb101,galera"
-
-ENV NAME=mariadb VERSION=10.1 RELEASE=1 ARCH=x86_64
```

Comment:
I removed it by mistake. Thank you for pointing this out.

It is added back.
(only for Fedora now - CentOS does not use `release` and `architecture` now. Also wouldn't it cause problems during rebuilding and automatic release bumping?)